README.md

```diff
@@ -401,6 +401,29 @@ private void OnMessage(DetailedResponse<MessageResponse> response, IBMError erro
 }
 ```
 
+## Transaction IDs
+Every SDK call will return a response which will contain a transaction ID, accessible via the `x-global-transaction-id` header. This transaction ID is useful for troubleshooting and accessing relevant logs from your service instance.
```

Comment:
@SirSpidey chimed in [here](https://github.com/watson-developer-cloud/dotnet-standard-sdk/pull/413) and offered updated messaging that better conforms to IBM's documentation style guide.

>Every SDK call returns a response with a transaction ID in the x-global-transaction-id header. This transaction ID is useful for troubleshooting and accessing relevant logs from your service instance.